helpers/service/apiService.js

```diff
@@ -0,0 +1,60 @@
+import axios from '../axios';
+import { HOST_API } from '../../config';
+
+const buildQuery = (query) => {
+  const newQuery = Object.keys(query)
+    .filter((key) => query[key])
+    .map((key) => `${key}=${query[key]}`)
+    .join('&');
+
+  return newQuery ? `?${newQuery}` : '';
+};
+
+const waitApiCallTime = 300000;
+
+export class ApiService {
+  constructor() {
+    if (!HOST_API) throw new Error('Please check .env config');
+    this.BASE_URL = `${HOST_API}`;
+  }
+
+  buildUrl(resource) {
+    return `${this.BASE_URL}/${resource}`;
+  }
+
+  get(resource, query = {}, config = {}) {
+    return axios.get(`${this.buildUrl(resource) + buildQuery(query)}`, {
+      headers: {
```

Comment:
Cors error is handled from backend, you can delete these lines of code for headers.